app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -165,7 +162,7 @@ class EventDetailsFragment : Fragment() {
 
             Picasso.get()
                     .load(eventViewModel.loadMap(event))
-                    .placeholder(R.drawable.ic_map_black_24dp)
+                    .placeholder(AppCompatResources.getDrawable(context!!, R.drawable.ic_map_black_24dp)!!)   //TODO: Make null safe
```

Comment:
@ayanovsk Please avoid all `!!` operators

Reply:
so what is the way to deal with it?
maybe you have the link where it is described?
I used cast before, it was also a problem.

Reply:
OK, let me try to fix it.

Reply:
wlll  something like so be a better option :         

                val myDrawable = (context?.let{AppCompatResources.getDrawable(it, R.drawable.ic_person_black_24dp)})?.let{it}

                myDrawable?.let { it1 ->
                    Picasso.get()
                            .load(it.avatarUrl)
                       //     .placeholder(R.drawable.ic_person_black_24dp)
                            .placeholder(it1)
                            .transform(CircleTransform())
                            .into(rootView.avatar)
                }

Reply:
No, I guess the above code will not be OK as either, as in case that real avatar exists, but placeholder does not, it will skip loading the real avatar ?

Reply:
Perhaps like this:

               val myDrawable = (context?.let { AppCompatResources.getDrawable(it, R.drawable.ic_person_black_24dp) })?.let { it }
                val firstPart =   Picasso.get()
                        .load(it.avatarUrl)
                (myDrawable?.let { 
                    firstPart.placeholder(it)
                } ?:   firstPart)
                        .transform(CircleTransform())
                        .into(rootView.avatar)
            }

Reply:
@dreadpool2 This is fine.

Reply:
Fine then 😄 @iamareebjamal . @ayanovsk, please go ahead 👍